docs/source/example_notebooks/dowhy_causal_discovery_example.ipynb

```diff
@@ -214,7 +355,7 @@
     "- Maximum in-degree: 3\n",
     "- Number of instances: 7466\n",
     "\n",
-    "The original causal graph is known for the Sachs dataset and we compare the original graph with the ones discovered using CDT in this section."
+    "The original causal graph is known for the Sachs dataset and we compare the original graph with the ones discovered using causal-learn in this section."
```

Comment:
the sachs dataset is still being loaded via CDT.  Rather than require installing all of CDT, can we just copy the load_sachs function, giving credit and following license requirements.  I believe it's a short function, though we'd also need an abbreviated version of the read_list_edges function.  https://github.com/FenTechSolutions/CausalDiscoveryToolbox/blob/master/cdt/data/loader.py#L87